File: cpp_e9zMZ/cpp/07-simdienthoai/93.cpp
#include <iostream>

#include "zvn.h"
using namespace std;

int main() {
    string phone = "0818999999";
    string result = "No";

    phone = myReplace(phone, ".");

    for (int i = 1; i < phone.length() - 5; i++) {
        char curr = phone[i];
        if (curr == phone[i + 1] && curr == phone[i + 2] && curr == phone[i + 3] && curr == phone[i + 4] && curr == phone[i + 5]) {
            result = "Yes";
            break;
        }
    }

    cout << result << endl;
    return 0;
}

File: python_ToSXs/python/07-simdienthoai/93.py
phone = "076.7777775"

phone = phone.replace(".", "")
result = "No"

for i in range(1, len(phone) - 5):
    if ((phone[i] == phone[i + 1]) and
            phone[i] == phone[i + 2] and
            phone[i] == phone[i + 3] and
            phone[i] == phone[i + 4] and
            phone[i] == phone[i + 5]):
        result = "Yes"
        break

print(result)
